services/mirror/mirror.go

```diff
@@ -438,6 +443,21 @@ func syncMirror(repoID string) {
 
 		// Create reference
 		if result.oldCommitID == gitShortEmptySha {
+			if tp == git.TagPrefix {
```

Comment:
It seems line 442 has done this?

Reply:
`SplitRefName` returns `refs/tags/` or `refs/heads/`, `NotifyCreateRef` uses `tag` and `branch` , for example: 
https://github.com/go-gitea/gitea/blob/d8c2c6cd59f3d38fcad9925fede4045c17975182/services/repository/push.go#L281-L290